nncf/quantization/algo.py

```diff
@@ -117,6 +117,11 @@ def __init__(self, quantizer_module_ref: BaseQuantizer,
 
 @COMPRESSION_ALGORITHMS.register('quantization')
 class QuantizationBuilder(CompressionAlgorithmBuilder):
+    DEFAULT_QUANTIZER_CONFIG = QuantizerConfig(bits=8,
+                                               mode=QuantizationMode.SYMMETRIC,
+                                               signedness_to_force=None,
```

Comment:
Didn't get what  `signedness_to_force` means? Not a user-friendly name, IMHO.

Reply:
This is an optional boolean value. Signedness to force = True meance force signed, False means force unsigned, None means do not force anything. Cannot make the name any more user-friendly than that while keeping its meaning.

Reply:
How about `force_signed=True`?

Reply:
Then `force_signed=False` becomes ambiguous - does it mean "do not force signed" or does it mean "force unsigned"?

Reply:
I introduced this `signedness_to_force` name a long time ago, and we have already discussed it then.

Reply:
Not to me at least, it looks like a command-line option I faced many times `--force-smthg`.